### ISSUE TITLE
[Feature Request] Pin favorites to the top of the list.

### ISSUE THREAD

**Issue by Amnesia1000** [citra-emu/citra#4910]
It would be good to be able to pin at the beginning our favorite games, as it can be done currently in Cemu. This will make it easier to find faster the games that users are currently playing. Even more if the user has several games in their list.

**Comment by vvanelslande** [comment]
You can do this on Nightly/Canary:
- Put your favorite games in a directory.
- Add it to Citra.
- Move it to the top.
- Expand it.

**Comment by Amnesia1000** [comment]
Yes, it is a possibility, but it is not only for the favorites, but also to have at hand those who are playing at the moment. And having to copy the games to another folder, then once finished take it out of that folder to the other and copy the new games that are going to start is tedious.
